utils/installer/install.sh

```diff
@@ -104,10 +112,6 @@ EOF
     echo "Would you like to install lunarvim's Rust dependencies?"
     read -p "[y]es or [n]o (default: no) : " -r answer
     [ "$answer" != "${answer#[Yy]}" ] && install_rust_deps
-  elif [ "$ARGS_INSTALL_NONINTERACTIVE" = "y" ]; then
```

Comment:
@kylo252: Shouldn't `--install-dependencies` always install the dependencies and not ask the user. 

I think it might be nice to still keep the non-interactive "install dependency" option like before.

Reply:
As I mentioned earlier, these dependencies are not that hard to manage from the outside, so I don't see the need to complicated things just for the sake of installing them.